### PR TITLE
Fixes #439 - test export / import of larger graphs to/from cypher

### LINKED ISSUE #439: test export / import of larger graphs to/from cypher
e.g. LDBC SF1

-  https://dl.dropboxusercontent.com/u/14493611/ldbc_sf001_p006.tgz

### export to cypher

- [x]  all
- [x] query
- [x] paths, nodes + rels
- [x] graph object

- [x] single file
- [x] many files
- [x] for the different output formats

###  import 
- [x] cypher-shell
- [x] -shell
- [x] apoc.cypher.runFile(s) , apoc.cypher.runSchemaFile(s)

### PR DESCRIPTION
**Pull request by DanielBerton**
We added the documentation with link to the issue and the commands that we used.

### CODE REVIEW

**Comment by DanielBerton** [comment]
We've created a script bash that run all the tests, if you want we can create also a `bat` for Windows

File: docs/big-graph-from-cypher.adoc [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+real    3m34.924s
+user    0m0.992s
+sys     0m0.068s
+----
+
+== Export from given nodes and rels
+
+[source,bash,subs=attributes]
+----
+time ./cypher-shell -u yourUsername -p yourPassword  "Match (n:Person)-[r:LIKES_COMMENT]->(c:Comment) CALL apoc.expor
```
Comment (jexp) [inline docs/big-graph-from-cypher.adoc]: this is wrong, it needs to collect the nodes and rels into 2 collections.
can you check the output file of this?

File: docs/big-graph-from-cypher.adoc [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+=== Result
+
+----
+real    0m0.683s
+user    0m0.896s
+sys     0m0.092s
+----
+
+== Import from file
```
Comment (jexp) [inline docs/big-graph-from-cypher.adoc]: where are the import measurements from file via `cypher-shell < file.cypher` and `neo4j-shell -file file.cypher` ?
those are more interesting than runFile (which should also fix).

**Comment by AngeloBusato** [comment]
@jexp  Done

**Pull request merged**